### PR TITLE
Support conf-libpcre2-8 on mingw

### PR DESCRIPTION
**Pull request by tobil4sk**
Modelled after #26893

### CODE REVIEW

File: packages/conf-libpcre2-8/conf-libpcre2-8.2/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
```
Comment (shonfeder) [inline packages/conf-libpcre2-8/conf-libpcre2-8.2/opam:2]: AFAIU, maintainer should name particular people (or organizations) who have
taken on the responsibility for maintaining a package. I don't think a link to
the opam repo issue tracker serves this end, and in any case it duplicates the
data we already have in the `bug-reports` field.

Would you be willing to maintain this package?

File: packages/conf-libpcre2-8/conf-libpcre2-8.2/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
```
Comment (shonfeder) [inline packages/conf-libpcre2-8/conf-libpcre2-8.2/opam:1]: What does the `-8` mean in the package name?

**Review by shonfeder — CHANGES_REQUESTED** [review]
Thank you for providing this system package! With the caveat that I don't have a lot of experience reviewing `conf-` packages yet, I don have one request and one question about the main opam file.

File: packages/conf-libpcre2-8/conf-libpcre2-8.2/opam (hunk 1 above)
Comment (tobil4sk) [reply packages/conf-libpcre2-8/conf-libpcre2-8.2/opam:2]: Hi, for `conf-libpcre2-8`, I am just updating the dependencies and the `pkg-config` check however there is already a previous version in the repository: `conf-libpcre2-8.1`. This previous version also put the issue tracker in place of the maintainer field. I'm not sure if I have the knowledge to maintain this package for every platform.

I feel more confident putting myself as maintainer for the newly added `conf-mingw-w64-pcre2-i686` and `conf-mingw-w64-pcre2-x86_64`.

File: packages/conf-libpcre2-8/conf-libpcre2-8.2/opam (hunk 2 above)
Comment (tobil4sk) [reply packages/conf-libpcre2-8/conf-libpcre2-8.2/opam:1]: There is an explanation for the library name in: #21349

File: packages/conf-libpcre2-8/conf-libpcre2-8.2/opam (hunk 1 above)
Comment (shonfeder) [reply packages/conf-libpcre2-8/conf-libpcre2-8.2/opam:2]: Oh, sorry I missed that this was just update! My comment is not relevant then.

File: packages/conf-libpcre2-8/conf-libpcre2-8.2/opam (hunk 2 above)
Comment (shonfeder) [reply packages/conf-libpcre2-8/conf-libpcre2-8.2/opam:1]: Got it, thanks! Sorry again that I missed the context of the previous release.

**Comment by shonfeder** [comment]
Only a single revdep failure from `pa_ppx_regexp` 0.02. I've confirmed (locally) that this is already failing to install with tests prior to this release.

Thanks again for your work packaging here!

**Review by shonfeder — APPROVED** [review]

**Pull request merged**